index.html

```diff
@@ -5,6 +5,26 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Donaate</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
```

Comment:
There are two body tags and navbar fix this

Reply:
meaning

Reply:
If you see the index.html file there is a body opening tag on line 13. But then there is a head closing tag on line 35 and then again a body opening tag.
- The code in between these lines is also repeated two times.

Reply:
Done you can check now